### DISCUSSION TITLE
Symfony trumbowyg colors plugin not working

### DISCUSSION THREAD

**Discussion by st33lsfx** [Alex-D/Trumbowyg#1296] [Q&A]
I work at Symfony. I have added Trumbowyg in the project. I added plugin colors, but nothing clicks when I click on the button. Colors are not displayed. I do not know what I am doing wrong. I have added a plugin and a css plugin. In btns: I have added ['backColor', 'foreColor']. The buttons appear but do nothing. thanks for the advice

V2.21.0

**Comment by Alex-D** [comment]
Without any code, nobody will be able to help you.

Are you using this bundle? Or anything else?
https://github.com/alexdw/trumbowygBundle
